CI Fails on install_chrome_for_tests

### Describe the current behavior

In actions, from time to time CI fails on install_chrome_for_tests

https://github.com/gristlabs/grist-core/actions/runs/13144405935/job/36679011839

```bash
Run buildtools/install_chrome_for_tests.sh -y
Reading package lists...
Building dependency tree...
Reading state information...
The following packages will be DOWNGRADED:
  google-chrome-stable
E: Packages were downgraded and -y was used without --allow-downgrades.
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 62 not upgraded.
Error: Process completed with exit code 100.
```

### Steps to reproduce

1. Run a CI

ex.
https://github.com/gristlabs/grist-core/actions/runs/13144405935/job/36679011839


### Describe the expected behavior

The expected version of chrome must install every time.
The bug let me think that there is undesired caching.

I'll open a PR in the next minutes to propose a fix.

### Where have you encountered this bug?

- On github CI

--- a/buildtools/install_chrome_for_tests.sh
+++ b/buildtools/install_chrome_for_tests.sh
@@ -19,6 +19,6 @@ if [[ "$(uname -m)" != "x86_64" ]]; then
 fi
 
 curl -sS -o /tmp/chrome.deb https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_${CHROME_VERSION}_amd64.deb \
-  && sudo apt-get install -y /tmp/chrome.deb \
+  && sudo apt-get install --allow-downgrades -y /tmp/chrome.deb \
   && rm /tmp/chrome.deb \
   && node_modules/selenium-webdriver/bin/linux/selenium-manager --driver chromedriver

